docs/management/admin/response-actions-history.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[response-actions-history]]
+= Response actions history
+
+{elastic-defend} keeps a log of the <<response-actions,response actions>> performed on endpoints, such as isolating a host or terminating a process. The log displays when each command was performed, the host on which the action was performed, the {kib} user who requested the action, any comments added to the action, and the action's current status.
+
+To access the response actions history for all endpoints, go to *Manage* -> *Response actions history*. You can also access the response actions history for an individual endpoint from these areas:
+
+* *Endpoints* page: Click an endpoint's name to open the details flyout, then click the *Response actions history* tab.
+* *Response console* page: Click the *Response actions history* button.
+
+All of these contexts contain the same information and features. The following image shows the *Response actions history* page for all endpoints:
+
+[role="screenshot"]
+image::images/response-actions-history-page.png[Response actions history page UI]
+
+Use these features to filter and expand the information in the response actions history:
```

Comment:
```suggestion
To filter and expand the information in the response actions history:
```

Reply:
Just a suggestion to further simplify the lead in sentence. I feel like you can remove the first few words in this sentence and still retain it's meaning and function. Or, you can keep it as is - I have no strong feelings one way or another.

Reply:
+💯 — I'm a big fan of editing for conciseness